src/activities/quizzes/QuizEntity.js

```diff
@@ -619,6 +619,18 @@ export class QuizEntity extends Entity {
 		return subEntity.hasClass(Classes.quizzes.attempts.hasAttempts);
 	}
 
+	hasNonAutoGradingQuestion() {
+		return this._entity && this._entity.properties && this._entity.properties.hasNonAutoGradingQuestion;
+	}
+
+	hasNonAutoGradingQuestionHref() {
```

Comment:
```suggestion
	nonAutoGradingQuestionHref() {
```
since it's returning the actual href, not bool.